appveyor.yml

```diff
@@ -24,18 +21,8 @@ build: false
 
 test_script:
     - ps: |
-        # We want to test dev version of the resource.
-        # xDscResouceDesigner works against standard PowerShell modules path.
-        # We need to make sure that root folder of xActiveDirectory is at the first place in $env:PSModulePath
-        if (($env:PSModulePath.Split(';') | select -First 1) -ne $pwd) {
-            $env:PSModulePath = "$pwd;$env:PSModulePath"
-        }
-        $testResultsFile = ".\TestsResults.xml"
-        $res = Invoke-Pester -OutputFormat NUnitXml -OutputFile $testResultsFile -PassThru
-        (New-Object 'System.Net.WebClient').UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path $testResultsFile))
-        if ($res.FailedCount -gt 0) {
-            throw "$($res.FailedCount) tests failed."
-        }
+        Invoke-AppveyorTestScriptTask `
+            -CodeCoverage
```

Comment:
We enabled CodeCov.io at the org level.  The missing code should be the only issue.